src/index.js

```diff
@@ -22,7 +22,7 @@ export function useElementSize() {
 }
 
 function getSizeFromEntry(entry) {
-  if (entry.borderBoxSize) {
+  if (entry.borderBoxSize && entry.borderBoxSize[Symbol.iterator] === 'function') {
```

Comment:
Instead of this check, implement a fallback using `contentRect` if borderBoxSize is not an array (`Array.isArray(...)`)

Reply:
@EECOLOR Heb hem zo aangepast, zodat er een fallback is wanneer `entry.contentRect` een `DOMRectReadOnly` is. Ik weet niet of die check ook echt expliciet nodig is, maar als `entry.borderBoxSize` niet ondersteunt is, had ik ook verwacht dat hij niet door de `if(entry.boderBoxSize)` zou komen, aangezien ik zou verwachten dat het `undefined` is.

Reply:
Ik snap je, dat dacht ik eerst ook, maar toen dacht ik: zou het misschien zo zijn dat ze in die versie achter de schermen dit hebben gedaan?

```
borderBoxSize = entry.contentRect
```